src/components/application_manager/test/policies/policy_handler_test.cc

```diff
@@ -120,15 +122,17 @@ class PolicyHandlerTest : public ::testing::Test {
   std::string default_hmi_;
   const std::string kPreloadPTFile_;
   const std::string kAppStorageFolder_;
-  ApplicationSet test_app;
-  sync_primitives::Lock app_lock;
-  DataAccessor<ApplicationSet> app_set;
+  ApplicationSet test_app_;
```

Comment:
@vlantonov Should be renamed as it's not just one app but the entire set of apps.